basicsr/models/archs/hifacegan_options.py

```diff
@@ -0,0 +1,156 @@
+import sys
+from basicsr.utils import Options
+
```

Comment:
I recommend merging this option to a YAML file, so that it could be consistent with the original repo.

Reply:
Guess I'll need to change every option call instead:) But let's do it your way.